homeassistant/components/renson/fan.py

```diff
@@ -41,6 +44,41 @@
     Level.LEVEL4.value: 4,
 }
 
+SET_TIMER_LEVEL_SCHEMA = {
+    vol.Required("timer_level", default="level1"): vol.In(
```

Comment:
A required with a default value is the same as an optional with a default value.
I would only add default value for the optional in the schema